client/src/components/Account/AccountBox.tsx

```diff
@@ -6,7 +6,7 @@ import FriendsTab from './Friends/FriendsTab';
 
 type Props = {};
 
-function AccountBox({}: Props) {
+function AccountBox() {
   const [currentTab, setCurrentTab] = useState<string | null>('Profile');
 
   const tabHandler = (tab: string | null) => {
```

Comment:
Similar to the previous comment ⬆️ these strings could be replaced with values from an enum.